packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md

```diff
@@ -36,6 +36,14 @@ This page provides the API rate limits for authentication and end-user activitie
 | `/api/{apiVersion}/radius`                                                                                             | 100              | 600              | 600     | 600        | 600                   |
 | `/idp/idx`                                                                                                             | 100              | 600              | 600     | 600        | 500                   |
 | `/login/token/redirect`                                                                                                             | 100              | 600              | 600     | 600        | 600                   |
+| <ApiLifecycle access="ie" /> **Identity Engine rate limit requests:**                                                                            |               |               |      |         |                    |
+| `/idp/idx`                                                                                                     | 100              | 600              | 600     | 600        | 1,200                   |
```

Comment:
I'd recommend re-ordering these rows to:
```
/idp/idx
/idp/idx/identify
/idp/idx/introspect
/idp/idx/challenge/poll
/idp/idx/authenticators/poll
/idp/idx/poll
```

Reply:
Fixed